src/frontends/tensorflow_common/src/op/gather.cpp

```diff
@@ -45,9 +63,35 @@ OutputVector translate_resource_gather_op(const NodeContext& node) {
 OutputVector translate_gather_v2_op(const NodeContext& node) {
     // GatherV2 has three inputs: data, indices, and axis by which data is sliced
     // batch_dims is an attribute and can vary
-    default_op_checks(node, 3, {"GatherV2"});
+    default_op_checks(node, 3, {"GatherV2"}, true);
+    auto params = node.get_input(0);
+    auto indices = node.get_input(1);
     auto axis = node.get_input(2);
     auto batch_dims = node.get_attribute<int64_t>("batch_dims", 0);
+
```

Comment:
`batch_dims` also needs to be updated if it is negative. And it should be passed to `Gather` operation creation